templates/download/raspberry-pi/thank-you.html

```diff
@@ -0,0 +1,79 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Thank you for downloading Ubuntu Server for Raspberry Pi{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1RXsLbMB-LJVwiG0gAi0HgVEJvZ9_GIU3ESLJ6KmBb54{% endblock meta_copydoc %}
+{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+
+{% block content %}
+<meta http-equiv="refresh" content="3;url=http://cdimage.ubuntu.com/releases/{{ version }}/release/ubuntu-{{ version }}-preinstalled-server-{{ architecture }}.img.xz">
```

Comment:
Just double checking the preinstalled-server is the correct download here?

Reply:
Yes.  These might change tonight.  But they are preinstalled, ready to be flashed onto a microSD card.